lib/web/mage/backend/validation.js

```diff
@@ -33,6 +33,17 @@
                     $(this.errorList.length && this.errorList[0].element || [])
                         .focus()
                         .trigger('focusin');
+                    $('[data-toggle=dropdown].active').trigger('open.dropdown');
+                    let first = true;
+                    $('.section-config').each(function(){
```

Comment:
Can you please explain what you try to do here?
For my point of view, all that we should do in this case it's find first element (that already did here `$(this.errorList.length && this.errorList[0].element || [])`) and check is this field inside of collapsible panel or not. If yes, check is collapsible panel is opened and if not - open this panel